Model/Method/Afterpay.php

```diff
@@ -1109,4 +1109,30 @@ protected function getIp($store)
 
         return $remoteAddress->getRemoteAddress();
     }
+
+    /**
+     * @param $response
+     *
+     * @return string
+     */
+    protected function getFailureMessage($response) {
+        if (!isset($response[0]) || empty($response[0])) {
+            return parent::getFailureMessage($response);
+        }
+        $transactionResponse = $response[0];
+        $responseCode        = $transactionResponse->Status->Code->Code;
+
+        if(!isset($transactionResponse->Services->Service->ResponseParameter->_)) {
+            return parent::getFailureMessage($response);
+        }
+
+        $message = $transactionResponse->Services->Service->ResponseParameter->_;
+        if(
```

Comment:
Format the code, space after 'if' 

if ($responseCode === 690
            && strpos($message, "deliveryCustomer.address.countryCode") !== false
        ) {